packages/cactus-core-api/src/main/typescript/plugin/plugin-aspect.ts

```diff
@@ -4,4 +4,5 @@ export enum PluginAspect {
   KV_STORAGE = "KV_STORAGE",
   WEB_SERVICE = "WEB_SERVICE",
   CONSORTIUM = "CONSORTIUM",
+  ATOMIC_SWAP = "ATOMIC_SWAP",
```

Comment:
@jordigiam Is this also present in the other branch? (the vanilla eth htlc). If yes, it's best to extract it to it's own PR.
Another thing: Would probably make sense to rename it to HTLC instead of ATOMIC_SWAP because what is getting clearer now is that the HTLC plugins themselves are a lower level of abstraction than the ATOMIC_SWAP plugins because the ATOMIC_SWAP plugins will manage cross-ledger flow and don't necessarily implement the swap via HTLCs either (could be a trusted relay or some escrow instead).

cc: @kikoncuo

Reply:
Done #587